Source/Core/Core/Core.cpp

```diff
@@ -240,9 +258,38 @@ void Stop()  // - Hammertime!
 	}
 }
 
+void DeclareAsCPUThread()
+{
+	#ifdef ThreadLocalStorage
+		tls_is_cpu_thread = true;
+	#else
+		//Use pthread implementation for Android
+		//Only initialize pthread keys once
+		#ifndef PTHREAD_IS_CPU_THREAD 
+			#define PTHREAD_IS_CPU_THREAD 
+			pthread_key_create(&s_tls_is_cpu_key, nullptr);
+		#endif
+
+		pthread_setspecific(s_tls_is_cpu_key, true);
```

Comment:
@magumagu I don't mean to sound stupid, but what is the best way to represent a boolean value as a void*?